addons/web_editor/static/tests/field_html_tests.js

```diff
@@ -132,6 +150,24 @@ QUnit.module('web_editor', {}, function () {
             form.destroy();
         });
 
+        QUnit.test('notebooks defined inside HTML field widgets are ignored when calling setLocalState', async function (assert) {
+            assert.expect(0);
+
+            var form = await testUtils.createView({
```

Comment:
for the next time, please try to use `const` everywhere and `let` where you can't (instead of `var`).

Reply:
it's a copy-pasted code :slightly_smiling_face: . Last time I was asked to change something in copy-pasted code and didn't end up well